Ginger/GingerCoreNET/RunLib/CLILib/CLIDynamicFile.cs

```diff
@@ -0,0 +1,183 @@
+                    {
+                        cliHelper.SourceControlProxyServer(exeConfiguration.SolutionScmDetails.ProxyServer);
+                        cliHelper.SourceControlProxyPort(exeConfiguration.SolutionScmDetails.ProxyPort);
+                    }
+                }
+                if (!string.IsNullOrEmpty(exeConfiguration.SolutionLocalPath))
+                {
+                    if (System.IO.Directory.Exists(exeConfiguration.SolutionLocalPath))
+                    {
+                        cliHelper.Solution = exeConfiguration.SolutionLocalPath;
+                    }
+                    else
+                    {
+                        Reporter.ToLog(eLogLevel.INFO, string.Format("Solution local path: '{0}' was not found so creating it", exeConfiguration.SolutionLocalPath));
+                        try
+                        {
+                            System.IO.Directory.CreateDirectory(exeConfiguration.SolutionLocalPath);
+                            cliHelper.Solution = exeConfiguration.SolutionLocalPath;
+                        }
+                        catch(Exception ex)
+                        {
+                            Reporter.ToLog(eLogLevel.ERROR, string.Format("Falied to create the Solution local path: '{0}'", exeConfiguration.SolutionLocalPath), ex);
+                        }
+                    }
+                    if (cliHelper.Solution == null)
+                    {
+
+                    }
+                }
+
+                if (exeConfiguration.ShowAutoRunWindow != null && exeConfiguration.ShowAutoRunWindow == true)
+                {
+                    cliHelper.ShowAutoRunWindow = true;
+                }
+                if (!string.IsNullOrEmpty(exeConfiguration.ArtifactsPath))
+                {
+                    cliHelper.TestArtifactsFolder = exeConfiguration.ArtifactsPath;
+                }                
+            }
+            else
+            {
+                //Dynamic XML
+                DynamicGingerExecution dynamicExecution = DynamicExecutionManager.LoadDynamicExecutionFromXML(content);
+                if (dynamicExecution.SolutionDetails.SourceControlDetails != null)
+                {
+                    cliHelper.SetSourceControlType(dynamicExecution.SolutionDetails.SourceControlDetails.Type);
+                    cliHelper.SetSourceControlURL(dynamicExecution.SolutionDetails.SourceControlDetails.Url);
+                    cliHelper.SetSourceControlUser(dynamicExecution.SolutionDetails.SourceControlDetails.User);
+                    cliHelper.SetSourceControlPassword(dynamicExecution.SolutionDetails.SourceControlDetails.Password);
+                    cliHelper.PasswordEncrypted(dynamicExecution.SolutionDetails.SourceControlDetails.PasswordEncrypted);
+                    if (string.IsNullOrEmpty(dynamicExecution.SolutionDetails.SourceControlDetails.ProxyServer) == false)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4812633)